src/main/java/com/isroot/stash/plugin/YaccServiceImpl.java

```diff
@@ -139,13 +139,29 @@ private boolean isCommitExcluded(Settings settings, YaccCommit commit) {
         return errors;
     }
 
+    private List<YaccError> checkCommiterEmailRegex(Settings settings, YaccCommit commit) {
+        List<YaccError> errors = Lists.newArrayList();
+        String regex = settings.getString("commiterEmailRegex");
```

Comment:
Can you fix typo in settings name so that setting name is correct when this is first deployed out? Committer needs another "t".